lib/matplotlib/legend.py

```diff
@@ -179,6 +179,8 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     also be made to match the color of the line or marker using 'linecolor',
     'markerfacecolor' (or 'mfc'), or 'markeredgecolor' (or 'mec').
 
+    Otherly, labelcolor can be set globally using :rc:`legend.labelcolor`.
+
```

Comment:
Sorry to be picky, bit this should go in as 
```
labelcolor : str or list, default: :rc:`legend.labelcolor`.
```

Reply:
Thanks for the review.
But when the `rcParams[legend.labelcolor]` is `None`(the default value(, should the docs say that it just uses the `rcParams['text.color']` value?

Reply:
I think that should be specified in the description of the parameter if it is not already

Reply:
This is fine w/ me now.  @tacaswell you have a block on this...